connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -866,4 +867,18 @@ public List<ConfigKeyInfo> connectorPluginConfig(String pluginName) {
         }
     }
 
+    @Override
+    public void connectorOffsets(String connName, Callback<ConnectorOffsets> cb) {
+        log.debug("Submitting offset fetch request for connector: {}", connName);
```

Comment:
Nit: these are all `TRACE` level in `DistributedHerder` (e.g., [here](https://github.com/apache/kafka/blob/6e8d0d9850b05fc1de0ceaf77834e68939f782c1/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java#L825), [here](https://github.com/apache/kafka/blob/6e8d0d9850b05fc1de0ceaf77834e68939f782c1/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java#L839), etc.)

```suggestion
        log.trace("Submitting offset fetch request for connector: {}", connName);
```